android-core/src/main/java/com/mparticle/networking/NetworkOptionsManager.java

```diff
@@ -11,23 +11,23 @@
 
 public class NetworkOptionsManager {
     public static String MP_CONFIG_URL = "config2.mparticle.com";
-    public static String MP_IDENTITY_URL = "identity.mparticle.com";
-    public static String MP_URL = "nativesdks.mparticle.com";
+    public static String MP_IDENTITY_URL = "identity";
```

Comment:
Nitpick but for clarity we should rename these to something like `MP_IDENTITY_SUBDOMAIN` or `MP_IDENTITY_URL_PREFIX` to clarify that it's not actually a URL. I did something similar on iOS